components/map/layers/shape.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Shape
+page_title: Map Layers - Shape
+description: Discover the Blazor Map Shape Layer and explore the examples.
+slug: components/map/layers/shape
+tags: telerik,blazor,map,layers,shape
+published: True
+position: 7
+---
+
+# Shape Layer
+
+You can bind, display and customize shapes with **Map** from JSON files in **GeoJSON** format.
```

Comment:
```suggestion
The shape layer provides binding to GeoJSON data and renders the specified geospatial objects.
```
Better not emphasize on JSON files as the data is not bindable to a file - as with other components, the origin of the data (file, database, etc.) is not important. The example itself can get the data from a file as this is one of the sources.